static/app/views/explore/hooks/useAnalytics.tsx

```diff
@@ -467,7 +467,7 @@ function computeConfidence(
   });
 }
 
-function computeTotals(
+export function computeTotals(
```

Comment:
If this is to be used elsewhere, move to a different file. We wouldn't be importing a function like this from analytics